File: PythonHost/Universal.py
def getHead(vLevel, vDivision):
	vResult = ""
	
	vResult += "<meta http-equiv=\"Content-Type\" content=\"text/html; charset=ISO-8859-1\">"
	vResult += getStyle(vDivision)
	
	return vResult
	
def getStyle(vDivision):
	vResult = ""
	
	vResult += "<link href=\"http://htkb.dyndns.org/Styles/"
	vResult += getStyleFile(vDivision)
	vResult += "\" rel=\"stylesheet\" type=\"text/CSS\">"
	
	return vResult
	
def getStyleFile(vDivision):
	vResult = ""
	vDefault = ""
	
	vDefault += "Main.css"

	if vDivision == "0":
		vResult = vDefault
	elif vDivision == "1":
		vResult = "Division1.css"
	elif vDivision == "2":
		vResult = "Division2.css"
	elif vDivision == "3":
		vResult = "Division3.css"
	elif vDivision == "4":
		vResult = "Division4.css"
	elif vDivision == "5":
		vResult = "Division5.css"
	elif vDivision == "6":
		vResult = "Division6.css"
	elif vDivision == "7":
		vResult = "Division7.css"
	else:
		vResult = vDefault
		
	return vResult

def getLogo():
	vResult = ""
	
	vResult += "<img id=\"idLogo\" src=\"http://htkb.dyndns.org/Pictures/logoHTKB.jpg\">"
	
	return vResult

def getNavBar():
	vResult = ""
	
	vResult += """
		<ul class=\"navBarUl\">
			<li><a class=\"navBar\" href=\"/Index\">Home</a></li>
			<li><a class=\"navBar\" href=\"/Division1/Index\">Board Games</a></li>
			<li><a class=\"navBar\" href=\"/Division2/Index\">Digital Board Games</a></li>
			<li><a class=\"navBar\" href=\"/Division3/Index\">Video Games</a></li>
			<li><a class=\"navBar\" href=\"/Division4/Index\">Downloadables</a></li>
			<li><a class=\"navBar\" href=\"/Division5/Index\">Database Work</a></li>
		</ul>
	"""
	
	return vResult

def getNavigationHeader():
	vResult = ""
	
	vResult += """
		<h4 class=\"headerColorBlack\">
			Navigation
		</h4>
	"""
	
	return vResult
	
def getInformationHeader():
	vResult = ""
	
	vResult += """
		<h4 class=\"headerColorBlack\">
			Information
		</h4>
	"""
	
	return vResult
	
def getInformation():
	vResult = ""
	
	vResult += """
		This is written with Web.py.
		Other versions can be found here:<br>
	"""
	
	return vResult
	
def getGDR():
	vResult = ""
	
	vResult += "<a href=\"http://htkb.dyndns.org/downloads/GDR.zip\">You can download my Games Development Report here.</a>"
	
	return vResult
	
def getWinRar():
	vResult = ""
	
	vResult += "<a href=\"http://htkb.dyndns.org/downloads/wrar371.exe\">You may need WinRar to open zip files from this site.</a>"
	
	return vResult
	
def getFooter():
	vResult = ""
	
	vResult += """
		<div id=\"idFooterContent\">
			Copyright 2012 All rights reserved<br>
			House That Kamurai Built<br>
		</div>
	"""
	
	return vResult
	
def getWebMaster():
	vResult = ""
	
	vResult += """
		<div id=\"idFooterManagement\">
			Website managed by Kamurai.
		</div>
	"""
	
	return vResult

File: ASPNetHost/Universal.js
function getHead(vLevel, vDivision){
	var vResult = "";
	
	vResult += "<head>";
		vResult += "<meta http-equiv=\"Content-Type\" content=\"text/html; charset=ISO-8859-1\">";
		vResult += "<link runat=\"server\" rel=\"shortcut icon\" href=\"/Pictures/favicon.ico\")\" type=\"image/x-icon\" />";
		vResult += "<link runat=\"server\" rel=\"icon\" href=\"/Pictures/favicon.ico\" type=\"image/ico\" />";
		vResult += getStyle(vLevel, vDivision);
	vResult += "</head>";

	return vResult;
}

function getStyle(vLevel, vDivision){
	var vResult = "";
	
	vResult += "<link href=\"http://htkb.dyndns.org/Styles/";
	vResult += getStyleFile(vDivision);
	vResult += "\" rel=\"stylesheet\" type=\"text/CSS\">";

	return vResult;
}

function getStyleFile(vDivision){
	var vResult = "";
	var vDefault = "";
	
	vDefault = "Main.css";
	
	if(vDivision == 0){
		vResult += vDefault;
	}else if(vDivision == 1){
		vResult += "Division1.css";
	}else if(vDivision == 2){
		vResult += "Division2.css";
	}else if(vDivision == 3){
		vResult += "Division3.css";
	}else if(vDivision == 4){
		vResult += "Division4.css";
	}else if(vDivision == 5){
		vResult += "Division5.css";
	}else if(vDivision == 6){
		vResult += "Division6.css";
	}else if(vDivision == 7){
		vResult += "Division7.css";
	}else{
		vResult += vDefault;
	}
	
	return vResult;
}

function getLogo(vLevel){
	var vResult = "";
	
    vResult += "<img id=\"idLogo\" src=\"http://htkb.dyndns.org/Pictures/logoHTKB.jpg\"><br>";
	
	return vResult;
}

function getNavBar(vLevel){
	var vResult = "";
	
    vResult += "<ul class=\"navBarUl\">";
		vResult += "<li><a class=\"navBar\" href=\"/ASPNET/Index.aspx\">Home</a></li>";
		vResult += "<li><a class=\"navBar\" href=\"/ASPNET/Division1/Index.aspx\">Board Games</a></li>";
		vResult += "<li><a class=\"navBar\" href=\"/ASPNET/Division2/Index.aspx\">Digital Board Games</a></li>";
		vResult += "<li><a class=\"navBar\" href=\"/ASPNET/Division3/Index.aspx\">Video Games</a></li>";
		vResult += "<li><a class=\"navBar\" href=\"/ASPNET/Division4/Index.aspx\">Downloadables</a></li>";
		vResult += "<li><a class=\"navBar\" href=\"/ASPNET/Division5/Index.aspx\">Database Work</a></li>";
	vResult += "</ul>";
		
	return vResult;
}

function getNavigationHeader(){
	var vResult = "";
	
    vResult += "<h4 class=\"headerColorBlack\">";
		vResult += "Navigation";
	vResult += "</h4>";
	
	return vResult;
}

function getInformationHeader(){
	var vResult = "";
	
    vResult += "<h4 class=\"headerColorBlack\">";
		vResult += "Information";
	vResult += "</h4>";
	
	return vResult;
}

function getInformation(){
	var vResult = "";
	
    vResult += "This is written with generic ASP.NET and javascript.<br><br>";
	vResult += "Other versions can be found here:<br>";
    
	return vResult;
}

function getGDR(){
	var vResult = "";
	
	vResult += "<a href=\"http://htkb.dyndns.org/downloads/GDR.zip\">You can download my Games Development Report here.</a></br>";
	
	return vResult;
}

function getWinRar(){
	var vResult = "";
	
	vResult += "<a href=\"http://htkb.dyndns.org/downloads/wrar371.exe\">You may need WinRar to open zip files from this site.</a></br>";	
	
	return vResult;
}

function getFooter(){
	var vResult = "";
	
	vResult += "<div id=\"idFooterContent\">";
		vResult += "© Copyright 2012 All rights reserved<br>";    
		vResult += "House That Kamurai Built<br>";
	vResult += "</div>";
	
	return vResult;
}

function getWebMaster(){
	var vResult = "";
	
    vResult += "<div id=\"idFooterManagement\">";
		vResult += "Website managed by Kamurai.";
	vResult += "</div>";
	
	return vResult;
}

function getPath(vLevel){
	var vResult = "";
	
    if(vLevel <= 0)
    {
        vResult += "./";
    }
    else if(vLevel == 1)
    {
        vResult += "../";
    }
    else if(vLevel == 2)
    {
        vResult += "../../";
    }
    else if(vLevel == 3)
    {
        vResult += "../../../";
    }
    else if(vLevel == 4)
    {
        vResult += "../../../../";
    }
    else if(vLevel == 5)
    {
        vResult += "../../../../../";
    }
    else if(vLevel == 6)
    {
        vResult += "../../../../../../";
    }
    else if(vLevel == 7)
    {
        vResult += "../../../../../../../";
    }
	
	return vResult;
}